src/main/java/sirius/biz/translations/BasicTranslations.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.translations;
+
+import sirius.db.mixing.BaseEntity;
+import sirius.db.mixing.Composite;
+import sirius.db.mixing.FieldLookupCache;
+import sirius.db.mixing.Mapping;
+import sirius.db.mixing.annotations.AfterDelete;
+import sirius.db.mixing.annotations.Transient;
+import sirius.kernel.di.std.ConfigValue;
+import sirius.kernel.di.std.Part;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * Provides database independent specifications for functionality to access and edit translations for an entity.
+ *
+ * @param <T> the translation entity type used by a concrete subclass
+ */
+public abstract class BasicTranslations<T extends BaseEntity<?> & Translation> extends Composite {
+
+    @Transient
+    protected BaseEntity<?> owner;
+
+    @Transient
+    @ConfigValue("mixing.multiLanguageStrings.supportedLanguages")
```

Comment:
maybe use a more generic config name here?

Reply:
this currently uses the existing supported languages config entry from the multi language strings implementation. It will be replaced once the standard codelists for countries/languages are implemented. The existing config should solve the chicken/egg-problem for now